src/Displaz.jl

```diff
@@ -234,28 +251,36 @@ When plotting lines, the `linebreak` keyword argument can be used to break the
 position array into multiple line segments.  Each index in the line break array
 is the initial index of a line segment.
 """
-function plot3d(plotobj::DisplazWindow, position; color=[1,1,1], markersize=[0.1], markershape=[0],
+function plot3d(plotobj::DisplazWindow, position; normal=[0,0,1], color=[1,1,1],
```

Comment:
Yes, a common fixed normal direction isn't much more helpful than no normal. I guess the only advantage would be to actually have a valid normal that isn't zero in length, which may be unexpected.

I would be interested in having a way to generate normals, but I'm not sure if that should go in here or directly into displaz (or neither)?